pkgs/applications/misc/megasync/default.nix

```
@@ -0,0 +1,143 @@
+{ lib
+, stdenv
+, autoconf
+, automake
+, bash
+, bashInteractive
+, c-ares
+, cryptopp
+, curl
+, doxygen
+, fetchFromGitHub
+, hicolor-icon-theme
+, libmediainfo
+, libraw
+, libsodium
+, libtool
+, libuv
+, libzen
+, lsb-release
+, makeDesktopItem
+, pkgconfig
+, qt5
+, sqlite
+, swig
+, unzip
+, wget
+, enableFFmpeg   ? true, ffmpeg  ? ffmpeg
```

```suggestion
, enableFFmpeg  ? true
, ffmpeg
```
ffmpeg does not need a default value (it is already the default package to be used).